app/javascript/packs/scroll.js

```diff
@@ -0,0 +1,12 @@
+window.addEventListener('scroll', () => {
+  const topbar = document.getElementsByClassName("top-bar")[0];
```

Comment:
why not use document.querySelector? in order not to select an array of elements. Applies container and topbar

Reply:
fixed